### DISCUSSION TITLE
Use a georeferenced file for map layer?

### DISCUSSION THREAD

**Discussion by jameslavery** [Mapsui/Mapsui#2184] [Q&A] [answered]
I'm investigating using Mapsui for a marine navigation app, and so need to be able to display raster charts. Is it possible to display georeferenced raster images as a layer?

**Comment by pauldendulk** [comment, marked as answer]
Yes it is. The image data is loaded in a `RasterFeature`. There are a few samples you could look at:
- *GeoTiff* under *Data Formats*. It uses a tfw georeference file.
- *Single Image* under *Data Formats*
- *Image over Map Data* under *Demo*
